ci-operator/step-registry/baremetalds/e2e/conf/router/baremetalds-e2e-conf-router-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think the portion of this PR related to the ovn-router has two problems:
1) The workflow is specific to one repo only (`ci-operator/config/openshift/router`) and not shared by many different repos (as it happens for the `ipv4/ipv6/virtualmedia/compact/dualstack` configurations)
2) The `test-list` approach it's one that we'd like to remove, as we're moving to run the whole suites for every job

Given that, I think it's better to remove this part from the current PR

Reply:
I'll remove the ovn-router and conf-router from this PR. Thanks.